examples/rtl_433_mqtt_hass.py

```diff
@@ -159,12 +159,29 @@
         }
     },
 
+    # This diagnostic sensor is useful to see when a device last sent a value,
+    # even if the value didn't change.
+    # https://community.home-assistant.io/t/send-metrics-to-influxdb-at-regular-intervals/9096
+    # https://github.com/home-assistant/frontend/discussions/13687
+    "time": {
+        "device_type": "sensor",
+        "object_suffix": "UTC",
+        "config": {
+            "device_class": "timestamp",
+            "name": "Timestamp",
+            "entity_category": "diagnostic",
+            "enabled_by_default": False,
+            "icon": "mdi:clock-in"
+        }
+    },
+
     "battery_ok": {
         "device_type": "sensor",
         "object_suffix": "B",
         "config": {
             "device_class": "battery",
             "name": "Battery",
+            "entity_category": "diagnostic",
```

Comment:
Both zwavejs and zigbee2mqtt mark batteries as diagnostic by default, so I've done the same here.